ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/ProtocolSpecBuilder.java

```diff
@@ -443,6 +456,10 @@ BlockValidator apply(
         Optional<GoQuorumPrivacyParameters> goQuorumPrivacyParameters);
   }
 
+  public interface WithdrawalsValidatorBuilder {
```

Comment:
Why do need to use a builder? Could just set the value directly on the ProtocolSpecBuilder

Reply:
I guess Jiri was following suit with how it works in the rest of the class

Reply:
I'll try direct, don't think it matters since there's no arguments to WithdrawalsValidator